packages/core/src/Synchronizer.ts

```diff
@@ -304,16 +331,15 @@ export class Synchronizer extends EventEmitter {
 
     private async _poll(): Promise<{ complete: boolean }> {
         if (!this.setupComplete) {
-            console.warn('polled before setup, nooping')
+            console.warn(
+                '@unirep/core:Synchronizer: polled before setup, nooping'
+            )
             return { complete: false }
         }
         this.emit('pollStart')
         const state = await this._db.findOne('SynchronizerState', {
-            where: {
-                attesterId: this.attesterId.toString(),
-            },
+            where: {},
```

Comment:
This seems ambiguous, shouldn't there be a filter? Otherwise it will return a document with unspecified ordering.